agent/engine/engine_windows_integ_test.go

```diff
@@ -127,6 +127,30 @@ func getContainerIP(client *docker.Client, id string) (string, error) {
 	return "", nil
 }
 
+func TestLocalHostVolumeMount(t *testing.T) {
+	cfg := defaultTestConfigIntegTest()
+	taskEngine, done, _ := setup(cfg, nil, t)
+	defer done()
+
+	// creates a task with local volume
+	testTask := createTestLocalVolumeMountTask("TestLocalHostVolumeMount")
+	stateChangeEvents := taskEngine.StateChangeEvents()
+	go taskEngine.AddTask(testTask)
+
+	verifyContainerRunningStateChange(t, taskEngine)
+	verifyTaskIsRunning(stateChangeEvents, testTask)
+	verifyContainerStoppedStateChange(t, taskEngine)
+	verifyTaskIsStopped(stateChangeEvents, testTask)
+
+	assert.NotNil(t, testTask.Containers[0].GetKnownExitCode(), "No exit code found")
+	assert.Equal(t, 0, *testTask.Containers[0].GetKnownExitCode(), "Wrong exit code")
+
+	assert.Contains(t, testTask.Volumes[0].Volume.SourcePath(), "C:\\ProgramData\\docker\\volumes")
+	data, err := ioutil.ReadFile(filepath.Join(testTask.Volumes[0].Volume.SourcePath(), "hello-from-container"))
+	assert.Nil(t, err, "Unexpected error")
+	assert.Equal(t, "empty-data-volume", strings.TrimSpace(string(data)), "Incorrect file contents")
```

Comment:
Im not sure how the Go libs work with ntfs here.. but will this opperate correctly if there is an EOF character?